chart/values.schema.json

```
@@ -799,8 +799,8 @@
             "x-docsSection": "Airflow",
             "default": null,
             "examples": [
-                "- secretRef:\n    name: '{{ .Release.Name }}-airflow-connections'",
-                "- configMapRef:\n    name: '{{ .Release.Name }}-airflow-variables'"
+                "- secretRef:\n    name: '{{ include 'airflow.fullname' . }}-airflow-connections'",
```

Even though it's an example, the quoting here seems like it'd be wrong. Maybe this?
```suggestion
                "- secretRef:\n    name: \"{{ include 'airflow.fullname' . }}-airflow-connections\"",
```